client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -577,7 +820,29 @@ class CancelPurchaseForm extends React.Component {
 	};
 
 	renderFreeformQuestion = () => {
-		const { translate } = this.props;
+		const { translate, isImport } = this.props;
+		const { surveyStep } = this.state;
+
+		if ( surveyStep === steps.FEEDBACK_STEP ) {
+			if ( ! isSurveyFilledIn( this.state, isImport ) ) {
+				// Do not display this question unless user has already answered previous questions.
+				return null;
+			}
+
+			return (
+				<div className="cancel-purchase-form__feedback-question">
+					<TextareaControl
+						label={ translate( "What's one thing we could have done better?" ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'What's one thing we could have done better? (optional)' )` *ES Score: 11*